inmanta.spec

```diff
@@ -33,15 +33,21 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildRequires:  systemd
 BuildRequires:  sed
 BuildRequires:  libffi-devel
-BuildRequires:  openssl-devel
 
 Requires:       git
 Requires:       gcc
-Requires:       openssl
 Requires:       logrotate
 Requires:       libffi
 Requires(pre):  shadow-utils
 
+%if 0%{?el7}
```

Comment:
why not: `if 0%{?rhel} == 7`?

Reply:
not sure if that works

Reply:
Same thing. I could change it if you want?

Reply:
does not really matter I guess.